code/items/ammo/BuckshotAmmo.cs

```diff
@@ -7,7 +7,7 @@ namespace TTTReborn.Items
     [Hammer.EditorModel("models/ammo/ammo_buckshot.vmdl")]
     partial class BuckshotAmmo : TTTAmmo
     {
-        public override string AmmoName => "buckshot";
+        public override string AmmoName => "ammo_buckshot";
```

Comment:
Just delete it and populate it in TTTAmmo on creation based on the LibaryName (like we did in the other entities) if not already done or not duplicated.